lib/processor.js

```diff
@@ -26,6 +27,52 @@ function getContentfulClient (accessToken, spaceId, host) {
 }
 
 /**
+ * Fetch common content for a certain space.
+ *
+ * @param {Object} options plugin config
+ * @param {Object} entries entries fetched from contentful
+ *
+ * @return {Object}        file mapping object with common content added
+ */
+function getCommonContentForSpace (options, entries) {
+  if (!options.common) {
+    return entries
+  }
+
+  const spaceId = options.space_id
+  const accessToken = options.access_token
+  const host = options.host
```

Comment:
i'm just going to use the options direclty in my call to `getContentfulClient` for now -- still think you're right about refactoring this though.